docs/preview/actions-runner-controller-2/README.md

```diff
@@ -0,0 +1,129 @@
+# Autoscaling Runner Scale Sets mode
+
+**⚠️ This mode is currently only available for a limited number of organizations.**
+
+This new autoscaling mode brings numerous enhancements (described in the following sections) that will make your experience more reliable and secure.
+
+## How it works
+
+![arc_hld_v1 drawio (1)](https://user-images.githubusercontent.com/568794/212665433-2d1f3d6e-0ba8-4f02-9d1b-27d00c49abd1.png)
+
+In addition to the increased reliability of the automatic scaling, we have worked on these improvements:
+
+- No longer require cert-manager as a prerequisite for installing actions-runner-controller
+- Reliable scale-up based on job demands and scale-down to zero runner pods
+- Reduce API requests to `api.github.com`, no more API rate-limiting problems
+- The GitHub Personal Access Token (PAT) or the GitHub App installation token is no longer passed to the runner pod for runner registration
+- Maximum flexibility for customizing your runner pod template
+
+### Demo
+
+https://user-images.githubusercontent.com/568794/212668313-8946ddc5-60c1-461f-a73e-27f5e8c75720.mp4
+
+## Setup
+
+### Prerequisites
+
+1. Create a K8s cluster, if not available.
+    - If you don't have a K8s cluster, you can install a local environment using minikube. See [installing Minikube](https://minikube.sigs.k8s.io/docs/start/).
+1. Install helm 3, if not available. See [installing Helm](https://helm.sh/docs/intro/install/).
+
+### Install actions-runner-controller
+
+1. Install actions-runner-controller using helm 3. For additional configuration options, see [values.yaml](https://github.com/actions/actions-runner-controller/blob/master/charts/actions-runner-controller-2/values.yaml)
+
+    ```bash
+    NAMESPACE="arc-systems"
+    helm install arc \
+        --namespace "${NAMESPACE}" \
+        --create-namespace \
+        oci://ghcr.io/actions/actions-runner-controller-charts/actions-runner-controller-2 \
+        --version 0.1.0
+    ```
+
+1. Generate a Personal Access Token (PAT) or create and install a GitHub App. See [Creating a personal access token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token) and [Creating a GitHub App](https://docs.github.com/en/developers/apps/creating-a-github-app).
+
+1. You're ready to install the autoscaling runner set. For additional configuration options, see [values.yaml](https://github.com/actions/actions-runner-controller/blob/master/charts/auto-scaling-runner-set/values.yaml)
+
+    ```bash
```

Comment:
A few things on this section:
- Can we split this in two blocks for easier copy paste? 😬 
- there's an extra `--set` on L55
- is it worth parameterising config url and pat/app details like we did for `NAMESPACE`? Makes it for cleaner, easier to mess around with commands especially if (ask me how I know) you get the config url wrong, much easier to edit a variable then a multi-line command
- the config url is a bit confusing, maybe worth specifying how to get one (i.e. from the new runner page)?
- should we point out that `arc-runner-set` is what will be used as the name of scale set? Helm newbie here had to figure that one out 🤦

Reply:
Great points! 

I already mentioned below that:
> The `runs-on` value should match the helm installation name you used in the previous step.

Probably here is a better spot

Reply:
I addressed these comments, can you have another check please?